js_modules/dagster-ui/packages/ui-core/src/nav/PipelineNav.tsx

```diff
@@ -50,7 +50,13 @@ export const PipelineNav = (props: Props) => {
   return (
     <>
       <PageHeader
-        title={<Heading>{pipelineName}</Heading>}
+        title={
+          <Heading style={{display: 'flex', flexDirection: 'row', gap: 4}}>
+            <a href="/jobs">Jobs</a>
```

Comment:
```suggestion
            <Link to="/jobs">Jobs</a>
```

Always use `<Link>` (from `react-router-dom`) for links within the app.